src/EditorFeatures/Core/ExternalAccess/VSTypeScript/Api/VSTypeScriptWaitContextWrapper.cs

```diff
@@ -4,18 +4,18 @@
 
 using System;
 using System.Threading;
-using Microsoft.VisualStudio.Utilities;
+using Microsoft.CodeAnalysis.Editor.Host;
 
 namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript.Api
 {
     [Obsolete("This is just a wrapper around the public Visual Studio API IUIThreadOperationContext, please use it directly.")]
```

Comment:
These obsolete comments probably need to be updated?

Reply:
They're still true; it's just that this is now wrapping the wrapper around IUIThreadOperationContext.